shopinvader_product_brand/models/shopinvader_brand.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ShopinvaderBrand(models.Model):
+    _name = "shopinvader.brand"
+    _description = "Shopinvader Brand"
+
+    _inherit = ["shopinvader.binding", "abstract.url", "seo.title.mixin"]
+    _inherits = {"product.brand": "record_id"}
+    _order = "sequence"
+
+    record_id = fields.Many2one(
+        "product.brand", required=True, ondelete="cascade", index=True
+    )
+    sequence = fields.Integer()
+    meta_description = fields.Char()
+    meta_keywords = fields.Char()
+    subtitle = fields.Char()
```

Comment:
subtitle?

Reply:
I was thinking to keep the wording of category but maybe a "slogan" or "headline" would be better, or maybe we can remove it and only add it in custom if needed